plugins/sfc/renderer/l2xconn/l2xconn_renderer.go

```diff
@@ -69,9 +70,9 @@ func (rndr *Renderer) AfterInit() error {
 
 // AddChain is called for a newly added service function chain.
 func (rndr *Renderer) AddChain(sfc *renderer.ContivSFC) error {
-	rndr.Log.Infof("Add SFC: %v", sfc)
+	rndr.Log.Infof("By Adel 1 - Add SFC: %v AfterSFC", sfc)
```

Comment:
Please get rid of the "By Adel" part in the log

Reply:
I have forget these local work comments on the committed version. I have deleted all personnel dev logs.